lib/components/TabView/TabContent.tsx

```diff
@@ -0,0 +1,24 @@
+import React, { FC } from "react";
+
+import styled from "styled-components";
+
+import { TabViewProps } from "./TabView.types";
+
+export const TabContentWrapper = styled.div<Pick<TabViewProps, "bgColor">>`
+  min-height: 100px;
+  padding: ${({ theme }) => theme.spacings.xbig};
+
+  border-radius: 0 0 10px 10px;
+
+  background-color: ${({ bgColor }) => bgColor};
```

Comment:
I don't think these are necessary, the client code can render whatever they want inside it, so it's best to leave it to them to style the content as they wish